src/PKSim.Presentation/Presenters/Charts/IndividualSimulationComparisonPresenter.cs

```diff
@@ -195,6 +195,7 @@ private void editChart(IndividualSimulationComparison chart)
       {
          InitializeAnalysis(chart);
          UpdateTemplatesBasedOn(_chartPresenterContext.ProjectRetriever.CurrentProject);
+         BindChartToEditors();
```

Comment:
This is the actual fix. Make sure we init the editors even if the comparison has NO simulations